### ISSUE TITLE
Content Wizard: Save action label is inconsistent

### ISSUE THREAD

**Issue by ashklianko** [enonic/app-contentstudio#8141]
Currently, the label alternates between 'Save' and 'Saved,' creating confusion about its intended use:

- When a save occurs automatically (e.g., due to page or site configuration updates), the label is 'Save.'
- After a manual save action, the label changes to 'Saved' and remains so, even for subsequent automatic save operations.

This behavior is misleading and distracting. Additionally, as users already receive a save confirmation notification at the bottom of the screen, the persistent 'Saved' label becomes redundant.

Update the 'Save' label behavior for all save actions:

- Display 'Saved' for a few seconds (matching the duration of the bottom notification).
- After this interval, revert the label to 'Save.'

This approach will provide better feedback without causing confusion or redundancy.

**Comment by sigdestad** [comment]
Hmm. Why not simply keep "saved" until someone changes it. Saved should always be disabled, while save should never be disabled, as it indicates the item is changed

**Comment by ashklianko** [comment]
What should we show initially when a content wizard is open? If it is a disabled 'Save' then after modifying and saving we're getting 'Saved', and no matter how we save the content, it stays 'Saved' always.

**Comment by sigdestad** [comment]
Do you mean read only mode?

**Comment by ashklianko** [comment]
No, I mean a regular mode, when a content is open for edit. We see this:

![image](https://github.com/user-attachments/assets/118c7ed3-abb9-439e-a614-b7958ec3d22d)

Then if we update some page component or site config then the content gets saved automatically and we get 'Save' again

![image](https://github.com/user-attachments/assets/1113477e-93fe-4f95-a918-4011060ccbfc)

However, if we manually update the content and save it ourselves, then we get 'Saved':

![image](https://github.com/user-attachments/assets/c513ad51-ad0f-49f9-95ae-a4e6fcbd1a0a)

And 'Saved' remains 'Saved' no matter how the content is saved after, automatically or manually

So in both cases the content was saved, but just one of them changes label to 'Saved'.

I believe that the bottom notification should be enough to notify that the content was saved, and we could use just 'Save' in the  action toolbar to avoid inconsistency. Or we can stick to using 'Saved' after ANY save, both options are better than now
